### ISSUE TITLE
Add support for installation via Homebrew

### ISSUE THREAD

**Issue by guygiat** [cyberark/conjur-api-python3#213]
## Is your feature request related to a problem? Please describe.

In order to improve the installation ux it would be great that MacOS users will be able to install the cli from homebrew.

## Describe the solution you would like

`brew install conjur-cli`

## Additional context

We already have a couple of homebrew formulas here: https://github.com/cyberark/homebrew-tools
We can add our cli to it.

**Comment by izgeri** [comment]
@sigalsax @InbalZilberman did you see this issue? wondering if there is a plan to add this support

**Comment by sigalsax** [comment]
@izgeri yes saw this and have been speaking to @guygiat about this (: I won't be for GA but I want to take this as an R&D boost task if not prioritized. @InbalZilberman where does this fall on priority?

**Comment by InbalZilberman** [comment]
I love the idea. We added it to the roadmap.
